DOC-1255 Clarify elements of the GDK in readme (#1287)
spatialos/UnrealGDK: README.md

![](SpatialGDK/Documentation/spatialos-gdkforunreal-header.png)

The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine fork and plugin with associated projects. It gives you the features of [SpatialOS](https://spatialos.improbable.io/docs/reference/latest), within the familiar workflows and APIs of Unreal Engine. For more information, please see the GDK's [documentation website](https://docs.improbable.io/unreal/latest).

> The SpatialOS GDK for Unreal is in alpha. It is ready to use for development of single-server games, but not recommended for public releases. We are committed to rapid development of the GDK to provide a performant release - for information on this, see our [development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) and [Unreal features support](https://docs.improbable.io/unreal/latest/unreal-features-support) pages, and contact us via our forums, or on Discord.

> This is the repository for the GDK plugin, which includes the Starter template (a blank starter project).

Reviewer: ```suggestion
This is the repository for the GDK plugin, which includes the Starter Template (a blank starter project).
```

> In addition to the plugin, the GDK also contains:

Reviewer: ```suggestion
In addition to the plugin, the GDK also includes:
```
Maybe "includes" because we are saying it's across three repos and "Contains" suggests there mught be a repo containing the GDK or something.... probs my bad coz I think I used "contains" somewhere.

> * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine)

Reviewer: ```suggestion
* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine)</br>
You must be a member of the [EpicGames organisation](https://github.com/EpicGames) on GitHub to follow this link. If you aren't, it returns a 404 error.
```

* [The Example Project](https://github.com/spatialos/UnrealGDKExampleProject)
  
If you’re an Unreal game developer and you’re ready to try out the GDK, follow the [Get started guide](https://docs.improbable.io/unreal/latest/content/get-started/introduction). 

> ## Unreal Engine changes

Reviewer: ```suggestion
## SpatialOS Unreal Engine fork changes
```
Make it consistent so easy to understand.

> In order to transform Unreal from a single server engine to a distributed model, we have made a number of small changes to the UE4 code. We will attempt to consolidate and remove (or submit as PR to Epic) as many of these changes as possible. You can see the changes in our forked [Unreal Engine repo](https://github.com/improbableio/UnrealEngine).

Reviewer: ```suggestion
In order to transform Unreal from a single-server engine to a distributed model, we made a number of small changes to Unreal Engine code. We are attempting to consolidate and remove (or submit as PR to Epic) as many of these changes as possible. You can see the changes in the [SpatialOS Unreal Engine fork repository](https://github.com/improbableio/UnrealEngine).
```

Made this up-to-date.
